src/main/java/org/prebid/server/handler/SetuidHandler.java

```diff
@@ -67,10 +79,14 @@ public void handle(RoutingContext context) {
         final String gdpr = context.request().getParam("gdpr");
         final String gdprConsent = context.request().getParam("gdpr_consent");
         final String ip = useGeoLocation ? HttpUtil.ipFrom(context.request()) : null;
-        gdprService.resultByVendor(GDPR_PURPOSES, gdprVendorIds, gdpr, gdprConsent, ip)
+        gdprService.resultByVendor(GDPR_PURPOSES, gdprVendorIds, gdpr, gdprConsent, ip, timeout(startTime))
```

Comment:
Since there are no time-consuming operations before this point timeout could be created just with `timeoutFactory.create(defaultTimeout)`, there is no need to trace start time from the very beginning